### ISSUE TITLE
Error with running sample app with cabal repl

### ISSUE THREAD

**Issue by F15Ky** [dmjio/miso#649]
**Describe the bug**
When I run the sample app with cabal repl I get the following errors:
```
Main.hs:26:17: error:
    Not in scope: data constructor ‘App’
    Perhaps you meant ‘Ap’ (imported from Miso.String)
   |
26 | main = startApp App {..}
   |                 ^^^

Main.hs:38:35: error:
    Not in scope: type constructor or class ‘Effect’
   |
38 | updateModel :: Action -> Model -> Effect Action Model
   |                                   ^^^^^^
Failed, no modules loaded.
```
It runs fine if I use nix-build with the sample default.nix 

**Desktop (please complete the following information):**
 - OS: Ubuntu with VirtualBox

**Comment by dmjio** [comment]
@F15Ky are you sure you're using `-fjsaddle` flag?

**Comment by F15Ky** [comment]
I get the same errors when running with -fjsaddle flag.

**Comment by dmjio** [comment]
This should be fixed now in `1.8`

**Comment by dmjio** [comment]
Re-open if this isn't fixed plz.
